jenkins/e2e-image/Makefile

```diff
@@ -18,7 +18,7 @@ TAG = $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)
 all: build
 
 kubetest:
-	GOOS=linux GOARCH=amd64 go build -o kubetest ../../kubetest
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o kubetest ../../kubetest
```

Comment:
my best guess would be sometime around the go1.5/1.6 timeframe when they switched the toolchain to go-native.